cookbook/playground/gemini_agents.py

```diff
@@ -0,0 +1,16 @@
+from phi.agent import Agent
+from phi.tools.yfinance import YFinanceTools
+from phi.playground import Playground, serve_playground_app
+from phi.model.google import Gemini
+
+finance_agent = Agent(
+    name="Finance Agent",
+    model=Gemini(id="gemini-2.0-flash-exp"),
+    tools=[YFinanceTools(stock_price=True)],
+    debug_mode=True,
+)
+
+app = Playground(agents=[finance_agent]).get_app(use_async=False)
```

Comment:
can we have use_async as true and the gemini model still working?
Cause openi, groq and other models work without turnning of `use_async`.

Reply:
@samyogdhital Gemini right now doesn't support async. It's on the roadmap but till then you have to use `use_async=False`